keras_contrib/optimizers/yogi.py

```diff
@@ -0,0 +1,92 @@
+from keras import backend as K
+from keras.optimizers import Optimizer
+from keras.utils.generic_utils import get_custom_objects
+
+
+class Yogi(Optimizer):
+    """Yogi optimizer.
+    Yogi is a variation of Adam that controls the increase in effective
+    learning rate, which (according to the paper) leads to even better
+    performance than Adam with similar theoretical guarantees on convergence.
+    Default parameters follow those provided in the original paper, Tab.1
+    # Arguments
+        lr: float >= 0. Learning rate.
+        beta_1: float, 0 < beta < 1. Generally close to 1.
+        beta_2: float, 0 < beta < 1. Generally close to 1.
+        epsilon: float >= 0. Fuzz factor. If `None`, defaults to `K.epsilon()`.
+        decay: float >= 0. Learning rate decay over each update.
+    # References
+        - [Adaptive Methods for Nonconvex Optimization](
+           https://papers.nips.cc/paper/8186-adaptive-methods-for-nonconvex-optimization)
+
+    If you open a issues or a pull request about the Yogi optimizer,
```

Comment:
an issue or a pull request

Reply:
Thanks for spotting that, seems I copy-pasted too enthusiastically. Will fix that in a minute!